tests/Integration/Behaviour/Features/Context/CategoryFeatureContext.php

```diff
@@ -0,0 +1,209 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace Tests\Integration\Behaviour\Features\Context;
+
+use Category;
+use Configuration;
+use Group;
+use RuntimeException;
+use Tests\Integration\Behaviour\Features\Context\Util\PrimitiveUtils;
+
+class CategoryFeatureContext extends AbstractPrestaShopFeatureContext
+{
+    /**
+     * @var int
+     */
+    private $defaultLanguageId;
+
+    public function __construct()
+    {
+        $this->defaultLanguageId = (int) Configuration::get('PS_LANG_DEFAULT');
+    }
+
+    /**
+     * @Given /^I specify "(.+)" "(.+)" for new category "(.+)"$/
```

Comment:
The phrase is too generic.
Feel free to improve, we need to understand what kind of value we can write. Pm don't know it must be a property, value and reference.

```suggestion
     * @Given /^I specify "(.+)" as property and "(.+)" as value for new category with reference"(.+)"$/
```

Reply:
Hmmmm, how about `@Given /^I specify property "(.+)" with value "(.+)" for new category "(.+)"` or `@Given /^I specify property "(.+)" and "(.+)" as its value for new category "(.+)"`?

Reply:
Look good :+1: !